bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -277,8 +277,8 @@ spec:
             {{- end }}
             {{- if or (.Files.Glob "files/my.cnf") .Values.mariadbConfiguration .Values.configurationConfigMap }}
             - name: mariadb-galera-config
-              mountPath: /bitnami/conf/my.cnf
-              subPath: my.cnf
+              mountPath: /bitnami/mariadb/conf/my_custom.cnf
```

Comment:
Could you try with` /bitnami/conf/my_custom.cnf`?

Reply:
Will give it a go @Mauraza, but the mysql_initialize script only looks for `my_custom.cnf` files within `${base_dir}/mariadb/conf/`. lemme see if it changes anything

Reply:
@Mauraza Nope, doesn't work - 

conf mounted at `/bitnami/conf/my_custom.cnf`

![image](https://user-images.githubusercontent.com/22311967/122054295-2a041100-ce05-11eb-87a0-5fa893c5e6be.png)


The main config being used by mariadb is - `/opt/bitnami/mariadb/conf/my.cnf`, and it has the default config - 

![image](https://user-images.githubusercontent.com/22311967/122054905-bc0c1980-ce05-11eb-851f-078c724844a7.png)


Ideally, the custom config should be copied to `/opt/bitnami/mariabd/conf/bitnami/my_custom.cnf` to be picked up the `!include` statement at the bottom. But, the folder empty - 

![image](https://user-images.githubusercontent.com/22311967/122055074-e1992300-ce05-11eb-9c9a-be62f579c559.png)


Looks like `/bitnami/mariadb/conf/my_custom.cnf` is the only one that works.